lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -116,7 +117,7 @@ serveWallet
     -- ^ HTTP API Server port.
     -> JormungandrBackend
     -- ^ Whether and how to launch or use the node backend.
-    -> (Port "wallet" -> Port "node" -> BlockchainParameters -> IO ())
```

Comment:
For minor convenience I remove `BlockchainParameters` form `beforeMainLoop` since it wasn't used.

Reply:
Maybe never mind.